tests/test_suite.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import e2e_logging
+import reconfiguration
+import recovery
+
+# For now, these are hardcoded.
+tests = [
+    reconfiguration.test_add_node,
+    reconfiguration.test_add_node_from_backup,
+    reconfiguration.test_add_as_many_pending_nodes,
+    reconfiguration.test_add_node_untrusted_code,
+    reconfiguration.test_retire_node,
+    e2e_logging.test,
+    e2e_logging.test_update_lua,
+    recovery.test,
+]
+
+#
```

Comment:
We could add a check for this with inspect.signature() or something similar, and make sure the calls all take two args with the right names. It's not perfect, but it should protect against accidents mostly.